tests/integration/Core/Repository/URLAliasServiceTest.php

```diff
@@ -1487,6 +1487,53 @@ public function testOverrideHistoryUrlAliasAtTheSameLocation(): void
         self::assertFalse($newAlias->isHistory);
     }
 
+    public function testRenamingParentContentDoesntBreakChildAlias(): void
+    {
+        $repository = $this->getRepository();
+        $urlAliasService = $repository->getURLAliasService();
+        $contentService = $repository->getContentService();
+
+        $languageCode = 'eng-GB';
+
+        // 1. Create parent folder
+        $folder = $this->createFolder([$languageCode => 'a'], 2);
+        $folderLocationId = $folder->contentInfo->getMainLocationId();
+
+        // 2. Create child folder
+        $child = $this->createFolder([$languageCode => 'b'], $folderLocationId);
+        /** @var \Ibexa\Contracts\Core\Repository\Values\Content\Location $childLocation */
+        $childLocation = $child->getVersionInfo()->getContentInfo()->getMainLocation();
```

Comment:
I see that this is a recent change.

Instead of using `@var` annotation, prefer doing a proper PHPUnit assertion.

```suggestion
        $childLocation = $child->getVersionInfo()->getContentInfo()->getMainLocation();
        self::assertNotNull($childLocation);
```
or
```suggestion
        $childLocation = $child->getVersionInfo()->getContentInfo()->getMainLocation();
        self::assertInstanceOf(Location::class, $childLocation);
```
(both result in the same type remaining in static analysis, as it's `Location|null` - so assertion limits the available type correctly)